safe_cli/operators/hw_accounts/ledger_manager.py

```diff
@@ -36,106 +30,30 @@ def connect(self) -> bool:
         if derivation_path[0:2] == "m/":
             derivation_path = derivation_path.replace("m/", "")
+        if LedgerManager.is_valid_derivation_path(derivation_path):
+            account = get_account_by_path(derivation_path)
+            return account.address
 
-        if not is_bip32_path(derivation_path):
-            raise InvalidDerivationPath()
-
-        account = get_account_by_path(derivation_path, self.dongle)
-        self.accounts.add(LedgerAccount(account.path, account.address))
-        return account.address
-
-    def delete_accounts(self, addresses: List[ChecksumAddress]) -> Set:
-        """
-        Remove ledger accounts from address
-
-        :param accounts:
-        :return: list with the delete accounts
-        """
-        accounts_to_remove = set()
-        for address in addresses:
-            for account in self.accounts:
-                if account.address == address:
-                    accounts_to_remove.add(account)
-        self.accounts = self.accounts.difference(accounts_to_remove)
-        return accounts_to_remove
-
-    @raise_as_hw_account_exception
-    def sign_eip712(self, safe_tx: SafeTx, accounts: List[LedgerAccount]) -> SafeTx:
-        """
-        Call ledger ethereum app method to sign eip712 hashes with a ledger account
-
-        :param domain_hash:
-        :param message_hash:
-        :param account: ledger account
-        :return: bytes of signature
-        """
-        encode_hash = eip712_encode(safe_tx.eip712_structured_data)
-        domain_hash = encode_hash[1]
-        message_hash = encode_hash[2]
-        for account in accounts:
-            print_formatted_text(
-                HTML(
-                    "<ansired>Make sure in your ledger before signing that domain_hash and message_hash are both correct</ansired>"
-                )
-            )
-            print_formatted_text(HTML(f"Domain_hash: <b>{domain_hash.hex()}</b>"))
-            print_formatted_text(HTML(f"Message_hash: <b>{message_hash.hex()}</b>"))
-            signed = sign_typed_data_draft(
-                domain_hash, message_hash, account.path, self.dongle
-            )
-
-            signature = signature_to_bytes(signed.v, signed.r, signed.s)
-            # TODO should be refactored on safe_eth_py function insert_signature_sorted
-            # Insert signature sorted
-            if account.address not in safe_tx.signers:
-                new_owners = safe_tx.signers + [account.address]
-                new_owner_pos = sorted(new_owners, key=lambda x: int(x, 16)).index(
-                    account.address
-                )
-                safe_tx.signatures = (
-                    safe_tx.signatures[: 65 * new_owner_pos]
-                    + signature
-                    + safe_tx.signatures[65 * new_owner_pos :]
-                )
+    @raise_ledger_exception_as_hw_account_exception
+    def sign_typed_hash(self, domain_hash, message_hash) -> bytes:
```

Comment:
- Missing types
- As `bytes` as a return type is not very declarative, could you add docs to say what you are returning and what are you receiving in the function?